### PR TITLE
feat(images): publish public docker images for mu/cu/su

### PR DESCRIPTION
**Pull request by dtfiedler**
Addressing https://github.com/permaweb/ao/issues/646

Dependent on https://github.com/permaweb/ao/pull/650

### CODE REVIEW

**Comment by dtfiedler** [comment]
These updated actions are demonstrated in https://github.com/ar-io/ao

**Comment by dtfiedler** [comment]
And the images can be used like so - https://github.com/ar-io/ao/pull/3

similar to ao-localnet - but doesn't include custom docker files

File: .github/workflows/cu.yml [hunk 1]
```diff
@@ -45,6 +45,38 @@ jobs:
...
+    permissions:
+      id-token: write
+      contents: read
+      checks: write
+      packages: write
+    steps:
+      - name: ⬇️ Checkout repo
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ needs.bump.outputs.TAG }}
```
[the remark this answers is not in the document]
Comment (dtfiedler) [reply .github/workflows/cu.yml]: removed in [70d6e86](https://github.com/permaweb/ao/pull/648/commits/70d6e862d60a296531a8f264be22708ecc75452a)

**Review by TillaTheHun0 — APPROVED** [review]
LGTM :shipit:

**Comment by dtfiedler** [comment]
One last thing @TillaTheHun0 - you'll need to make the images `public` in the `packages` section of the repository. Related docs- https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility

**Pull request merged**